platform/cosmos/api.go

```diff
@@ -91,6 +91,15 @@ func (p *Platform) GetValidators() (blockatlas.ValidatorPage, error) {
 	return results, nil
 }
 
+func (p *Platform) GetDetails() blockatlas.StakingDetails {
+	//TODO: Find a way to have a dynamic
+	return blockatlas.StakingDetails{
+		Reward:        blockatlas.StakingReward{Annual: 11},
+		MinimumAmount: blockatlas.Amount("0"),
+		LockTime:      1814400,
```

Comment:
Lets do it dynamic? we can do like that:
curl -X GET "https://stargate.cosmos.network/staking/parameters" -H "accept: application/json"
Field: unbonding_time

Reply:
It's dynamic for delegations API,

it's only static for blockchain details itself, not per delegation

Reply:
Why not to do it dynamic too?